### PR TITLE
address terraform 0.12.16 deprecation warnings

### PR DESCRIPTION
**Pull request by jamieklassen**
far fewer quotes, dollars and braces. looks prettier.

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -70,7 +70,7 @@ module "cluster" {
       auto-upgrade = false
       disk-size    = "50"
       disk-type    = "pd-ssd"
-      image        = "COS"
+      image        = "UBUNTU"
```
Comment (cirocosta) [inline terraform/main.tf:73]: 🤔 is this intended?

Comment (cirocosta) [reply terraform/main.tf:73]: (as part of "address terraform 0.12.16 deprecation warnings")

Comment (deniseyu) [reply terraform/main.tf:73]: looks like this PR built on top of #82

**Review by deniseyu — APPROVED** [review]

File: terraform/main.tf (hunk 1 above)
Comment (jamieklassen) [reply terraform/main.tf:73]: Yup shoulda rebased

**Review by cirocosta — APPROVED** [review]

**Pull request merged**